commitizen/cli.py

```diff
@@ -183,6 +183,13 @@
                         "default": False,
                         "help": "retry commit if it fails the 1st time",
                     },
+                    {
+                        "name": "manual_version",
```

Comment:
Should we make it `--manual-version`? 🤔

Reply:
I would not mind, but believe that makes no practical difference: 

1. decli/argparse normalises the argument name, i.e. bump.py uses `manual_version = self.arguments["manual_version"]` either way.
2. `cz bump --help` does not display the argument name, but the metavar value, i.e. `MANUAL_VERSION`.

Let me know, if you want me to change the PR.

Reply:
I prefer it like the proposal from this PR, as it is inline with the way changelog works: `cz changelog 1.2.3` would generate the changelog for that version. I would expect `cz bump 1.2.3` to bump the given version as well.

Reply:
If that's the case, let's go with it 👍